project/AppDependencies.scala

```diff
@@ -2,7 +2,7 @@ import play.sbt.PlayImport._
 import sbt._
 
 object AppDependencies {
-  val bootStrapPlayVersion = "5.18.0"
+  val bootStrapPlayVersion = "5.24.0"
```

Comment:
There is a later version of 5.25.0 for this (avoiding 6.0.0 and later) Should we probably use that ?

Reply:
Updated - now resolved